src/aiida_ase/calculations/ase.py

```diff
@@ -239,7 +239,7 @@ def return_a_function(v):
         input_txt += f"atoms = ase.io.read('{self._input_aseatoms}')\n"
         input_txt += '\n'
         input_txt += f'calculator = custom_calculator({calc_argsstr})\n'
-        input_txt += 'atoms.set_calculator(calculator)\n'
+        input_txt += 'atoms.calc = calculator\n'
```

Comment:
Why is this change necessary?

Reply:
I was getting a warning that was being pulled into the results saying that it should be set like this now. I think it was still working but producing the warning.